test/integ_tests/test_simulator_quantum_task.py

```diff
@@ -46,34 +46,36 @@
 
 # shots-based tests in this file have the capacity to fail rarely due to probabilistic checks.
 # this parameter can be adjusted if we find tests regularly failing.
-SHOTS = 9000
+SHOTS = 2000
```

Comment:
Is there a risk of numerical discrepancies causing tests to fail?

Reply:
Raised to 4500 to mitigate this.